platform/linuxbsd/display_server_x11.cpp

```diff
@@ -1205,10 +1205,7 @@ float DisplayServerX11::screen_get_refresh_rate(int p_screen) const {
 bool DisplayServerX11::screen_is_touchscreen(int p_screen) const {
```

Comment:
I guess this method should be removed until and if it's actually implemented (a comment does not have any useful information).

Reply:
Done. Also removed in DisplayServerWindows:
```
virtual bool screen_is_touchscreen(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
virtual void screen_set_orientation(ScreenOrientation p_orientation, int p_screen = SCREEN_OF_MAIN_WINDOW) override;
virtual ScreenOrientation screen_get_orientation(int p_screen = SCREEN_OF_MAIN_WINDOW) const override;
```

We should probably have an issue that keeps track of all the methods in DisplayServer and their implementation status in each platform, so we know which ones are TODOs and which ones are left out because not relevant.

Reply:
In the case of Windows, this can be done in general with `GetSystemMetrics(SM_DIGITIZER);`, but associating a touch input device with a specific display won't be easy. Probably will involve enumerating and matching sub-strings in the raw HID and display device IDs (if it's always have matching part, it which I'm not sure of).